app/views/devise/confirmations/new.html.erb

```diff
@@ -13,4 +13,4 @@
   </div>
 <% end %>
 
-<%= render "devise/shared/links" %>
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] Great job working on the devise setup but I think it would be great if you fix the confirmation so when the user signup he needs to confirm his email first since after signing up it redirect me to the user index page I could see you add :confirmable to the user model but I think you need to add in user controller `before_action :authenticate_user!` , please check this [article](https://reintech.io/blog/how-to-authenticate-users-in-rails) for more information 

and this should not be this case the user needs to confirm his email first then login with the mail and password 🔥♻💪🏻 so kindly add this as it is mentioned in the requirements 🔥👏🏻

![image](https://github.com/Ochiengsteven/art-post/assets/58553711/58f092a3-1331-408f-8933-8ff63481bc9e)